controller/test-files/search/csv/ok.res.payload.golden.json

```diff
@@ -0,0 +1 @@
+"Area,Assignees,Boardcolumns,Codebase,Created at,Creator,Description,Effort,Execution Order,Iteration,Labels,Meta State,Number,Remote item,State,Title,Updated at\n,,,,0001-01-01T00:00:00Z,John Doe 00000000-0000-0000-0000-000000000001,`(see function github.com/fabric8-services/fabric8-wit/controller_test.(*searchControllerTestSuite).TestSearchWorkItemsCSV.func2 in controller/search_blackbox_test.go)`,42.000000,1000.000000,,important;backend,,1,,New,work item 00000000-0000-0000-0000-000000000002,0001-01-01T00:00:00Z\n"
```

Comment:
Thank you for adding this golden file. I wonder if we can have the `\n` appear as a real line break.

Reply:
Also, please rename the file to `ok.res.payload.golden.csv`.

Reply:
I see the name is fixed in 12b2693 . Thanks.